_data/projects/accessibility-insights-web.yml

```
@@ -0,0 +1,11 @@
+name: Accessibility Insights for Web
+desc: Accessibility Insights for Web is a Google Chrome and Microsoft Edge Insider browser extension for assessing the accessibility of web sites and web applications.
+site: https://accessibilityinsights.io
+tags:
+  - chromeextension
```

Just one suggestion to make this a bit easier to read:

```suggestion
  - chrome-extension
```

---

updated. Somehow i read in the instructions that `-` is not allowed. Thanks for pointing this out.